.github/workflows/ci.yml

```diff
@@ -25,19 +25,25 @@ jobs:
       image: ${{ matrix.docker_image }}
     steps:
     - uses: actions/checkout@v2
-    - uses: ros-tooling/setup-ros@0.1.2
+    - uses: ros-tooling/setup-ros@master
       with:
         required-ros-distributions: ${{ matrix.ros_distribution }}
+        use-ros2-testing: true
     - name : Download and install rclc-dependencies
       run: |
         apt-get install ros-${{ matrix.ros_distribution }}-osrf-testing-tools-cpp
         apt-get install ros-${{ matrix.ros_distribution }}-test-msgs
-    - uses : ros-tooling/action-ros-ci@0.1.0
+    - uses : ros-tooling/action-ros-ci@master
       with:
         package-name: "rclc rclc_examples rclc_lifecycle"
         vcs-repo-file-url: dependencies.repos
         target-ros2-distro: ${{ matrix.ros_distribution }}
-        colcon-mixin-name: coverage-gcc
+        colcon-defaults: |
+        {
+          "build": {
+            "mixin": [ "coverage-gcc" ]
+          }
+        }
```

Comment:
the indentation is wrong here. The 5 lines above should be indented by two spaces more.